src/trimgalore/script.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -eo pipefail
+
+[[ ! -d $output_dir ]] && mkdir -p $par_output_dir
+
+IFS=";" read -ra input <<< $par_input
+
+unset_if_false=( 
```

Comment:
All these should become `boolean_true` IMO. See also above. Can you please check?